tests/platform_tests/mellanox/check_hw_mgmt_service.py

```diff
@@ -13,10 +13,6 @@ def check_hw_management_service(dut):
     assert hw_mgmt_service_state["ActiveState"] == "active", "The hw-management service is not active"
     assert hw_mgmt_service_state["SubState"] == "exited", "The hw-management service is not exited"
 
-    logging.info("Check thermal control status")
```

Comment:
I remember we have another PR to handle this? suggest not to add this in this PR.